src/Less/index.less

```diff
@@ -208,11 +225,11 @@ a[href=''] {
 }
 
 .cv-hl-in {
-  background: linear-gradient(to left, #FFC264, white);
+  #gradient.horizontal(white, #FFC264);
```

Comment:
Mostly just a nifty shorthand, but it does also add vendor prefixing: https://github.com/Niharika29/PlagiabotWeb/pull/12/files#diff-f7cc4de34a9083a20c1c95fd3725b03eR9